docs/semgrep-ci/manual-ci-setup.md

```diff
@@ -0,0 +1,415 @@
+          script:
+            - export SEMGREP_RULES="p/default" # TODO get confirmation from Holden
+            - semgrep ci # TODO confirm that this will work
+```
+
+### CircleCI
+
+To add Semgrep into your CircleCI pipeline, do the following steps.
+
+1. Create or edit a `config.yml` configuration file to add a Semgrep command as
+   part of your CircleCI workflow. Refer to the [CircleCI code snippet](#circleci-code-snippet).
+2. Optional: You can also review the [CirclCI YAML introduction](https://circleci.com/docs/introduction-to-yaml-configurations) to help with editing the configuration file.
+3. Commit the configuration file into the `/.circleci` folder within the target
+   repository.
+4. The Semgrep job starts automatically upon detecting the `config.yml` commit in the previous step.
+5. Customize the job's behavior, such as its rules to scan and files to ignore.
+
+#### CircleCI code snippet
+
+```yaml
+version: 2.1
+jobs:
+  semgrep-scan:
+    parameters:
+      repo_path:
+        type: string
+		    # Change This to your repository path.
+			  # Sara: Isn't there a way to fetch these values from Circle CI so it's not hardcoded?
+			  # TODO Check with Holden.
+        default: myorg/semgrep-test-repo
+      default_branch:
+        type: string
+        default: main
+    environment:
+      # Scan changed files in PRs, only report new findings (existing findings ignored)
+      SEMGREP_BASELINE_REF: << parameters.default_branch >>
+      SEMGREP_RULES: p/default
+	
+    # Change job timeout (in seconds; default is 1800 seconds. Set to 0 to disable)
+    #   SEMGREP_TIMEOUT: 300
+
+    docker:
+      - image: returntocorp/semgrep
+    steps:
+      - checkout
+      - run:
+          name: "Semgrep scan"
+          command: semgrep ci
+workflows:
+  main:
+    jobs:
+      - semgrep-scan
+```
+
+### Buildkite
+
+To add Semgrep into your CircleCI pipeline, do the following steps.
+
+1. Create or edit a `pipelines.yml` configuration file to add a Semgrep command as part of your pipeline. Refer to the [BuildKite code snippet](#buildkite-code-snippet). This configuration file can also be stored within Buildkite.
+2. Optional: You can also review the [Buildkite steps definition](https://buildkite.com/docs/pipelines/defining-steps) to help with editing the configuration file.
+3. If you are using Buildkite to store the configuration, save the file. Otherwise, commit the configuration file into the `/.buildkite` folder within the target repository.
+4. The Semgrep job starts automatically upon detecting the `pipelines.yml` commit in the previous step.
+5. Customize the job's behavior, such as its rules to scan and files to ignore.
+
+:::note
+From Buildkite's main page, click **Pipelines > ➕ button** to perform these steps within Buildkite's UI.
+:::
+
+#### Buildkite code snippet
+
+```yaml
+- label: ":semgrep: Semgrep"
+  commands:
+    - export SEMGREP_RULES='p/default' # to confirm with Holden that this will work
```

Comment:
@holdenmcg One of the things we're trying to do is use p/default to expose the SEMGREP_RULES value, so that it's easy for folks to either migrate to semgrep app (no need to edit semgrep ci --config auto) or add more rulesets. Will the above command work with buildkite?

Reply:
Tried this just now, works like a charm! 👍